src/app/components/Figure/Caption/index.jsx

```diff
@@ -0,0 +1,11 @@
+import styled from 'styled-components';
+import { FF_NEWS_SANS_REG } from '../../../lib/constants/styles';
+
+const Caption = styled.figcaption`
+  background-color: #d5d0cd;
+  color: #404040;
```

Comment:
This should be `{$C_STORM}` https://github.com/bbc/simorgh/blob/data-schema-validator-part-2/src/app/lib/constants/styles.js#L7

Reply:
https://github.com/bbc/simorgh/issues/322